abis_mapping/templates/survey_site_data/schema.json

```diff
@@ -45,6 +56,212 @@
         "required": false
       }
     },
+    {
+      "name": "landformPattern",
+      "title": "Land Form Pattern",
+      "description": "A description of landform at a broad scale (best mapped at 1:200,000) based on slope, morphological type, dimensions, mode of geomorphological activity and geomorphological agent.",
+      "example": "Alluvial fan",
+      "type": "string",
+      "format": "default",
+      "constraints": {
+        "required": false,
+        "enum": [
+          "ALLUVIAL FAN",
+          "ALLUVIAL PLAIN",
+          "ANASTOMOTIC PLAIN",
+          "BADLANDS",
+          "BAR PLAIN",
+          "BEACH RIDGE PLAIN",
+          "CALDERA",
+          "CHENIER PLAIN",
+          "CORAL REEF",
+          "COVERED PLAIN",
+          "DELTA",
+          "DUNEFIELD",
+          "ESCARPMENT",
+          "FLOODPLAIN",
+          "HILLS",
+          "KARST",
+          "LACUSTRINE PLAIN",
+          "LAVA PLAIN",
+          "LONGITUDINAL DUNEFIELD",
+          "LOW HILLS",
+          "MADE LAND",
+          "MARINE PLAIN",
+          "MEANDER PLAIN",
+          "METEOR CRATER",
+          "MOUNTAINS",
+          "PARABOLIC DUNEFIELD",
+          "PEDIMENT",
+          "PEDIPLAIN",
+          "PENEPLAIN",
+          "PLAIN",
+          "PLATEAU",
+          "PLAYA PLAIN",
+          "RISE",
+          "SAND PLAIN",
+          "SHEET-FLOOD FAN",
+          "STAGNANT ALLUVIAL PLAIN",
+          "TERRACE",
+          "TERRACED LAND",
+          "TIDAL FLAT",
+          "VOLCANO"
+      ]
+      }
+    },
+    {
+      "name": "landformElement",
+      "title": "Landform Element",
+      "description": "A description of landform at a finer scale (best mapped at 1:15,000) based on slope, morphological type, dimensions, mode of geomorphological activity and geomorphological agent.",
+      "type": "string",
```

Comment:
should be `list`

Reply:
I'm getting: 

`frictionless.exception.FrictionlessException: [schema-error] Schema is not valid: descriptor is not valid (Field is not valid: constraint "enum" is not supported by type "list")`

Reply:
interesting test case

Reply:
the `list` field plugin is likely going to need to be updated to handle enums. It can be tied into the string field type's handling of enums hopefully.

Reply:
ok i think what might have to happen is append "enum" to the `ListField.supported_constraints` list attribute. Then overload the definition of [`check_enum`](https://github.com/frictionlessdata/frictionless-py/blob/75be5990da82965e7c02f3d5309c1579e34f84c3/frictionless/schema/field.py#L338) to perform checks on a per element basis.

Reply:
as discussed please set to list and remove enum constraint. At the same time please capture this in a backlog ticket to be rectified.